src/osp/UserInputHandler.h

```diff
@@ -348,23 +339,63 @@ class UserInputHandler
 
 private:
 
-    std::vector<ButtonMap> m_deviceToButtonRaw;
-    std::map<std::string, ButtonConfig> m_controlConfigs;
-    std::vector<ButtonControl> m_controls;
+    std::vector<ButtonMap_t> m_deviceToButtonRaw;
+    std::map<std::string, ButtonControlConfig, std::less<> > m_btnControlCfg;
```

Comment:
Good opportunity for shared string.

Reply:
Not quite sure if this is the case. Strings here are matched with other strings, and no other class would really need to hold references to the strings stored in here.